hub.go

```diff
@@ -738,6 +739,18 @@ func HubWithWebSocketConnection() HubOption {
 	}
 }
 
+// HubWithSenderMaxRetryCount configures the Hub to only retry sending messages `maxRetryCount` times.
+func HubWithSenderMaxRetryCount(maxRetryCount int) HubOption {
+	return func(h *Hub) error {
+		if h.senderRetryOptions == nil {
+			h.senderRetryOptions = newSenderRetryOptions()
```

Comment:
Could this be moved into the constructor for `Hub`?  Then you don't need to call it in more than one place.

Reply:
There are two spots where a Hub is constructed, so I think this could work. I'll shift this code over.

Reply:
Done.